src/components/GrowerDetail.js

```diff
@@ -106,6 +140,23 @@ const GrowerDetail = (props) => {
                   }))
                   .filter((id) => id),
               );
+              let statusCount = emptyStatusCount;
+              registrations.map((reg) => {
```

Comment:
These stats are meant to be capture (tree) counts, not grower (planter) registration counts.

Reply:
Thanks for the feedback. I will make the changes